.github/workflows/cd.yaml

```diff
@@ -0,0 +1,83 @@
+name: HOPR CD
+
+on:
+  push:
+    branches: ["debug/**"]
+    paths-ignore:
+      - "docs/**/*"
+      - "packages/*/docs/**/*"
+
+jobs:
+  cd:
+    name: Build HOPRD
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git info
+        run: ./scripts/configure-git-info.sh
+
+      - name: Setup Google Cloud Credentials
+        uses: google-github-actions/setup-gcloud@v0.2.1
+        with:
+          project_id: ${{ secrets.GOOGLE_HOPRASSOCIATION_PROJECT }}
+          service_account_key: ${{ secrets.GOOGLE_HOPRASSOCIATION_CREDENTIALS_REGISTRY }}
+          export_default_credentials: true
+
+      - name: Set image version
+        id: set-image-version
+        run: echo "::set-output name=vsn::$(date +%s)"
+
+      - name: Set package version
+        id: set-package-version
+        run: echo "::set-output name=vsn::$(./scripts/get-package-version.sh)"
+        env:
+          HOPR_PACKAGE: hoprd
```

Comment:
Can be removed, its now the first parameter of the script.  And the default is already `hoprd`.